src/connectors/breadcrumb/__tests__/connectBreadcrumb-test.ts

```diff
@@ -135,12 +143,13 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/breadcrumb/
     it('returns the `SearchParameters` with another `hierarchicalFacets` already defined', () => {
       const render = () => {};
       const makeWidget = connectBreadcrumb(render);
-      const helper = jsHelper({}, '', {
+      const helper = jsHelper(createSearchClient(), '', {
         hierarchicalFacets: [
           {
             name: 'country',
             attributes: ['country', 'sub_country'],
             separator: ' > ',
+            // @ts-ignore
```

Comment:
why did this one need to be ignored? value seems valid to me.

Reply:
Actually, `rootPath`, as specified on the helper, can't be null. `null !== undefined`. Any ideas?

Reply:
we should probably use `undefined` instead of `null` OR update the types then

Reply:
Done.